lib/tasks/multitenant/tenants.rake

```diff
@@ -9,5 +9,15 @@ namespace :multitenant do
         end
       end
     end
+
+    task suspend_forbidden_plans_scheduled_for_deletion: :environment do
+      return unless Features::AccountDeletionConfig.enabled?
```

Comment:
For the rake task, better adding an output message when it's disabled, like "Account deletion is disabled. Nothing to do.".